src/redux/book/bookSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const BooksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    AddBook: (state, action) => {
+      state.books.push(action.payload);
+    },
+    RemoveBook: (state, action) => {
+      state.books = state.books.filter((book) => book.id !== action.payload);
+    },
```

Comment:
## Optional Change ♻️
- [ ] It's a good convention to have our reducer function names start with a lowercase letter. For instance, the AddBook should be written as `addBook`. Kindly modify your reducer function names 👍🏽

Reply:
you got to be kidding me bruh.

Reply:
It's not a required change though, but it's a good convention to start a function name in lowercase letters. 👍🏽

Reply:
👉🏼 removeBook: (state, action) => {
      state.books = state.books.filter((book) => book.id !== action.payload);
    },